src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/recordings/test_azconfig_mgmt.yaml

```diff
@@ -513,7 +513,7 @@ interactions:
       - python/3.5.2 (Windows-10-10.0.18362-SP0) msrest/0.6.10 msrest_azure/0.6.2
         azure-mgmt-keyvault/1.1.0 Azure-SDK-For-Python AZURECLI/2.0.81
```

Comment:
It seems that you are still using `azure-mgmt-keyvault/1.1.0`, actually the new version (2.1.1) of this SDK had already been released, and it was bumped in the Private Link PR.

Reply:
I don't want to re-run those tests other than keyvault. There are so many pitfalls and manual intervention needed in the live run. Also the request header isn't considered in the vcr matcher.

Reply:
I have the same concern.  And it looks in azure-mgmt-keyvault 1.1.0 there is no v2019-09-01 SDKs.

Reply:
> I don't want to re-run those tests other than keyvault. There are so many pitfalls and manual intervention needed in the live run. Also the request header isn't considered in the vcr matcher.

Yes, I think it's not necessary to re-run all tests, but my concern is: are you using `1.1.0` for recording? If yes, you should better upgrade it to `2.1.1` because this version was already written in pip requirements, all newly modified yaml should use the latest packages for the consistency concern.

Reply:
I didn't re-run the test live. The recording YAML is manually edited. I couldn't run `azdev` on all tests as it hangs forever. I couldn't even find out which test hangs.

Reply:
Thanks, got it.

Reply:
@Juliehzl, I have live run the storage test `test_customer_managed_key` and it works well. The recording now shows `azure-mgmt-keyvault/2.1.1`:

https://github.com/Azure/azure-cli/blob/d09618e9669a097fde3c50fa3ac0ea0785b4053c/src/azure-cli/azure/cli/command_modules/storage/tests/latest/recordings/test_customer_managed_key.yaml#L128-L130

Kindly review it.